cert-expiry-finder

```diff
@@ -294,10 +294,29 @@ expiry_after() {
     esac
 }
 
+list_k8s_cert_expiry_days() {
+    local name namespace notafter unixtime delta context
+    if ! which kubectl; then
+        return
+    fi
+    kubectl config get-contexts --no-headers -oname | while read context
+    do
+        kubectl get cert --context $context -A -ojson \
```

Comment:
`"$context"`

I don't know if it's allowed to have a space, but might as well take it into consideration.

And I prefer the `=` if allowed (which it is): `--context="$context"`

Reply:
changed